packages/meditrak-server/src/routes/exportSurveyResponses.js

```diff
@@ -7,9 +7,14 @@ import xlsx from 'xlsx';
 import moment from 'moment';
 import fs from 'fs';
 import { truncateString } from 'sussol-utilities';
-import { DatabaseError, ValidationError } from '@tupaia/utils';
+import {
+  DatabaseError,
+  ValidationError,
```

Comment:
Please remove ValidationError as it's not used (it was already here before the PR but might as well clean it up)